src/tablo/Tablo.d.ts

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+
+export type HeaderProps = {
+  children: any
+};
+
+export type FooterProps = {
+  children: any
+}
+
+export type ColumnGroupProps = {
+  fixed?: boolean,
+  children?: any // TODO(typo) it should be Columns[]
+};
+
+export type CellProps<Data> = {
+  children?: any | ((data: any , rowData: Data, rowIndex: number) => JSX.Element),
+  backgroundColor?: React.CSSProperties['backgroundColor'],
+  color?: React.CSSProperties['color'],
+  vAlignContent?: 'top' | 'center' | 'bottom',
+  hAlignContent?: 'left' | 'center' | 'right',
+  contentStyle?: React.CSSProperties,
+  style?: React.CSSProperties
+};
+
+export type ColumnProps<Columns> = {
+  name: Columns,
+  width?: number,
+  fixed?: boolean,
+  isResizable?: boolean,
+  flexGrow?: number,
+  children?: any, // TODO(typo) it should be (Header | Cell | Footer)[],
+  allowCellsRecycling?: boolean
+};
+
+type SelectedRows = number[];
+type SortBy<Columns> = Columns;
+type SortDir = 'asc' | 'desc';
+
+export type TabloProps<Data, Columns> = {
+  data: Data[],
+  className?: string,
+  rowHeight?: number,
+  headerHeight?: number,
+  groupHeaderHeight?: number,
+  footerHeight?: number,
+  onRowMouseEnter?: (rowIndex: number) => void,
+  onRowMouseLeave?: (rowIndex: number) => void,
+  columnsOrder?: Columns[],
+  onColumnsReorder?: (columns: string[]) => void,
+  onColumnResize?: (x: { width: number, key: string}) => void,
```

Comment:
already changed, using directly `Columns` since it's ideally an enum (the list of the columns of the table), but I typed it as `string` in the export.